subsys/bluetooth/host/classic/sdp.c

```diff
@@ -1750,6 +1750,11 @@ static int sdp_client_receive(struct bt_l2cap_chan *chan, struct net_buf *buf)
 
 	switch (hdr->op_code) {
 	case BT_SDP_SVC_SEARCH_ATTR_RSP:
+		/* Check the buffer len for the length field */
+		if (buf->len < 2) {
```

Comment:
Just for consistency, I suppose `sizeof(uint16_t)` would be better than `2`. Otherwise looks good.

Reply:
Good. I changed it.